docs/source/software/quick_start_guide.rst

```diff
@@ -50,17 +50,12 @@ Companion Computer Setup
 ========================
 
 You will need to run a similar setup on the companion computer (CC). For
-simplicity, we have prepared disk images that can be cloned via Clonezilla
-straight onto new boards. They currently come in three different flavours:
+simplicity, we have prepared disk images that can be cloned via Etcher
+straight onto new boards. We currently use the ODriod UX4 as our companion
+computer.
 
-1. Odroid
-2. Upboard
-3. TX2
-
-.. TODO:
-    Link to cloned image for those not in lab.
-
-Borrow the relevant USB stick from the lab.
+Instructions for flashing can be found
+:ref:`here<software/Companion_computer:Companion Computer>`.
```

Comment:
i think you need a newline here? Before `:ref:`

Reply:
I will double check this and see what's going on. My guess, is I missed the correct spelling/capitalization of the link. When you saw this, was it with your local build? Did you see any errors or warnings in the output?

Reply:
I currently cannot reproduce this on my machine.

It seems to me this is a Sphinx environment issue... I added the extension `sphinx.ext.autosectionlabel` in order to generate labels for all the section headers automagically. This required a change to the `conf.py` and most likely will require a `make clean && make html` locally in order to see the correct functionality. But let me know if you've done this and are still seeing the issue. If so, there still may be something I'm doing on my machine that allows this to function as I was hoping and I haven't provided it to the repo.

Reply:
seems resolved now. super minor by 
`Image of the OS (kessel.stanford.edu/media/data/odroid_images/20190305_MSL_Quad_ODroidKernel4_14_Ubuntu16_04_Rootless_GPIO.img`

extends off the page. change to 
the image of the OS can be found on `kessel` at 
`Image of the OS (kessel.stanford.edu/media/data/odroid_images/`
with the file name
`20190305_MSL_Quad_ODroidKernel4_14_Ubuntu16_04_Rootless_GPIO.img`

Reply:
Great catch. Updating this and then merging!